package.json

```diff
@@ -25,15 +26,24 @@
     "test": "npm-run-all --parallel test:unit test:functional",
     "token": "node scripts/token.js"
   },
-  "pre-commit": [
-    "precommit-msg",
-    "format",
-    "lint"
-  ],
-  "pre-push": [
-    "prepush-msg",
-    "test"
-  ],
+  "lint-staged": {
+    "./*.{js,jsx}": [
+      "eslint --cache --fix",
+      "git add"
+    ],
+    "./{src,test,scripts}/**/*.{js,jsx}": [
+      "eslint --cache --fix",
+      "git add"
+    ],
+    "./*.{js,jsx,html}": [
+      "prettier --write",
+      "git add"
+    ],
+    "./{src,test,scripts}/**/*.{js,jsx,html}": [
+      "prettier --write",
+      "git add"
+    ]
+  },
```

Comment:
We didn't have lint-staged set up previously. This runs linting and formatting on staged files during a commit.